src/adapters/marketplace/v00.js

```diff
@@ -39,10 +39,15 @@ class V00_MarkeplaceAdapter {
 
   async createListing(
     ipfsBytes,
-    { deposit = '0', arbitrator },
+    { deposit = '0', arbitrator, commission },
     confirmationCallback
   ) {
     const from = await this.contractService.currentAccount()
+    deposit = commission && commission.amount || '0'
```

Comment:
Paranoia: should we check commission.currency==='OGN' and add a comment that in V1 we only support commission in that currency ?